projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_common.g4

```diff
@@ -293,6 +293,7 @@ port
    | KSHELL
    | LDAP
    | LDAPS
+   | LDP
```

Comment:
Need to add `else if` case to `org.batfish.grammar.cisco.CiscoControlPlaneExtractor:toNamedPort`. Then you may need to add data model `NamedPort`.

Reply:
done